packages/app/src/App.tsx

```diff
@@ -71,6 +71,7 @@ const App = () => {
             <LocalizationProvider customMessages={CUSTOM_MESSAGES}>
               <Layout
                 navigationItems={menuItems}
+                menuPollingInterval={config.MENU_POLLING_INTERVAL}
```

Comment:
Please don't drill down this var thru the whole app. But import the config file in MenuItem

Reply:
I tried doing that, of course it's bad to drill, but the improting failed due to the config sitting in the implementation app and not the libraries themselves.
I'll have another look at this.

Reply:
I had another look with Niels and we came to the conclusion that this is the only way without adding a root app component that handles the setting of global config items.

This isn't even drilling since we only provide a value to the "base" component of nl-portal and then set it into a context for other components in the libs to make use of.

I will leave it as is and we can adress how to set the config globally in another task/story (that I will write).